### PR TITLE
Banner ui

### PR DESCRIPTION
**Pull request by georgiashay**
the hacky things with emitting cookie events between MainPage and Auth will go away when Vuex is merged

adds colored banners and automatically allows cookies if you edit your road

### CODE REVIEW

File: src/pages/MainPage.vue [hunk 1]
```diff
@@ -373,7 +375,9 @@ export default {
     roads: {
       handler: function (newRoads, oldRoads) {
         this.justLoaded = false;
-        if (this.activeRoad !== '') {
+        this.cookiesAllowed = true;
+        this.$refs.authcomponent.allowCookies();
+        if (this.activeRoad != '') {
```
Comment (npfoss) [inline src/pages/MainPage.vue]: is this supposed to be a single `=`?

Comment (georgiashay) [reply src/pages/MainPage.vue]: No, I think this happened because this branch started before the linter.  Will fix when I have time to get to my computer.

**Comment by georgiashay** [comment]
Deployed to dev and noticed a bug caused by the main page vs auth cookies.  Not deploying to prod until fixed (if you opt out then change road, it will update cookies to true - should not do this)

**Comment by npfoss** [comment]
Ah good catch, thanks for being so thorough

**Comment by georgiashay** [comment]
Fixed in #235

**Pull request merged**